src/js/services/configService.js

```diff
@@ -68,6 +68,21 @@ angular.module('copayApp.services').factory('configService', function(storageSer
       enabled: false,
     },
 
+    blockExplorerServices: [
```

Comment:
the service listing  suggest that adding a service is just adding a new item in the config, and it is not.